external-crates/move/crates/move-analyzer/editors/code/scripts/create.sh

```diff
@@ -64,29 +63,29 @@ rm -rf $LANG_SERVER_DIR
 mkdir $LANG_SERVER_DIR
 
 for DIST_OS VSCODE_OS in "${(@kv)SUPPORTED_OS}"; do
-    # Iota distribution identifier
-    IOTA_DISTRO=$NETWORK"-v"$VERSION
-    # name of the Iota distribution archive file, for example iota-testnet-v1.0.0-macos-arm64.tgz
-    IOTA_ARCHIVE="iota-"$IOTA_DISTRO"-"$DIST_OS".tgz"
+    # IOTA distribution identifier
+    IOTA_VERSION="v"$VERSION
+    # name of the Iota distribution archive file, for example iota-v1.0.0-macos-arm64.tgz
+    IOTA_ARCHIVE="iota-"$IOTA_VERSION"-"$DIST_OS".tgz"
     # a path to downloaded Iota archive
     IOTA_ARCHIVE_PATH=$TMP_DIR"/"$IOTA_ARCHIVE
 
-    # download Iota archive file to a given location and uncompress it
-    curl https://github.com/iotaledger/iota/releases/download/"$IOTA_DISTRO"/"$IOTA_ARCHIVE" -L -o $IOTA_ARCHIVE_PATH
+    # download IOTA archive file to a given location and uncompress it
+    curl https://github.com/iotaledger/iota/releases/download/"$IOTA_VERSION"/"$IOTA_ARCHIVE" -L -o $IOTA_ARCHIVE_PATH
     tar -xf $IOTA_ARCHIVE_PATH --directory $TMP_DIR
 
     # names of the move-analyzer binary, both the one becoming part of the extension ($SERVER_BIN)
     # and the one in the Iota archive ($ARCHIVE_SERVER_BIN)
     SERVER_BIN="move-analyzer"
-    ARCHIVE_SERVER_BIN=$SERVER_BIN"-"$DIST_OS
+    ARCHIVE_SERVER_BIN=$SERVER_BIN
     if [[ "$DIST_OS" == *"windows"* ]]; then
         SERVER_BIN="$SERVER_BIN".exe
         ARCHIVE_SERVER_BIN="$ARCHIVE_SERVER_BIN".exe
     fi
 
     # copy move-analyzer binary to the appropriate location where it's picked up when bundling the
     # extension
-    SRC_SERVER_BIN_LOC=$TMP_DIR"/external-crates/move/target/release/"$ARCHIVE_SERVER_BIN
```

Comment:
Already part of `TMP_DIR`?

Reply:
Can you please elaborate on the question? I don't understand what you mean

Reply:
This scripts downloads them from a release, and there the `move-analyzer` bin is not in this sub directory